components/images/images-en.html

```diff
@@ -0,0 +1,249 @@
+</dl>
+
+<h3>Responsive images</h3>
+<p>Use to make larger images scale properly, and never exceed the width of the parent container. With responsive images, <code>max-width: 100%;</code> and <code>height: auto;</code> are auto-applied with the use of <code>.img-responsive.</code></p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<p>Image is too large for the available space: </p>
+				<img src="https://placehold.it/400x100" alt="Generic placeholder image">
+				<p class="mrgn-tp-lg">Same image is now responsive, and fits the parent container:</p>
+				<img src="https://placehold.it/400x100" class="img-responsive" alt="Generic placeholder image">
+			</div>
+		</div>
+	</div>
+</div>
+
+<h4>Source code</h4>
+<pre><code>&lt;img src=&quot;https://placehold.it/400x100&quot; class=&quot;img-responsive&quot; alt=&quot;Generic placeholder image&quot;&gt;</code></pre>
+
+<h3>Stretching images</h3>
+<p>Use to stretch a smaller image to the width of the container. Ensure the image is still clear and easy to see in the larger resolutions.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<p>Image (100x25)  is too small  and blurry: </p>
+				<img src="https://placehold.it/100x25" alt="Generic placeholder image">
+				<p class="mrgn-tp-md">Image (200x50) is still too small for the space:</p>
+				<img src="https://placehold.it/200x50" alt="Generic placeholder image">
+				<p class="mrgn-tp-lg">Stretched image (100x25) is blurry:</p>
+				<img src="https://placehold.it/100x25" class="full-width" alt="Generic placeholder image">
+				<p class="mrgn-tp-lg">Stretched, clear image (200x50)  fits space: </p>
+				<img src="https://placehold.it/200x50" class="full-width" alt="A generic square placeholder image">
+			</div>
+		</div>
+	</div>
+</div>
+
+<h4>Source code</h4>
+<pre><code>&lt;img src=&quot;https://placehold.it/200x50&quot; class=&quot;full-width&quot; alt=&quot;A generic square placeholder image&quot;&gt;</code></pre>
+
+<h3>Thumbnail tiles</h3>
+<p>Use to add any kind of content like headings, paragraphs, or buttons into a thumbnail, to create a thumbnail tile effect.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<figure class="thumbnail"><a href="#"><img src="https://placehold.it/350x200" alt="Generic placeholder thumbnail"></a>
+					<figcaption class="caption">
+						<h4>Title (caption) </h4>
+						<p>Content</p>
+						<ul class="list-inline">
+							<li><a href="#" class="btn btn-primary" role="button">Button</a></li>
+							<li><a href="#" class="btn btn-default" role="button">Button</a></li>
+						</ul>
+					</figcaption>
+				</figure>
```

Comment:
This is a bad example which are not compliant with info & relationship.

Can we think of another example or the same but coded differently

Reply:
Fixed